### PR TITLE
Create KliKli-milestone_1.md

### PR DESCRIPTION
**Pull request by mg00736448**
upload milestone

# Milestone Delivery Checklist

- [x] The [milestone-delivery-template.md](https://github.com/crustio/Crust-Grant-Milestone-Delivery/blob/master/deliveries/milestone-delivery-template.md) has been copied and updated.
- [x] The [invoice form :pencil:](https://forms.gle/d7FQS6tg27Rc9K7SA) has been filled out for this milestone. 
- [x] This pull request is being made by the same account as the accepted application. 
- [ ] In case of acceptance, the payment will be transferred to the initial ETH(USDT/CRU) payment address. 
- [ ] The delivery is according to the [Guidelines for Milestone Deliverables](https://github.com/crustio/Crust-Grant-Milestone-Delivery/blob/main/milestone-deliverables-guidelines.md).

### CODE REVIEW

File: deliveries/KliKli-milestone_1.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Milestone Delivery
+
+> Only the GitHub account, which is responsible for the pull request of the accepted application is allowed to submit milestones. 
+
+Before submit this milestone delivery, please make sure: **The [invoice form](https://docs.google.com/forms/d/e/1FAIpQLSfxKTRtoMzvqQiBL71YXA6gxl_XSsyNBHEBksFFC2AwWReU1w/viewform?usp=sf_link) has been filled out correctly for this milestone and the delivery is according to the official [milestone delivery guidelines](milestone-deliverables-guidelines.md).**  
+
+Please provide `PR Link` and `Milestone Number` at following.
+* **PR Link:** Provide a link to the initial accepted pull request of your application to the [Crust Grants Program repository](https://github.com/crustio/Crust-Grants-Program).
```
Comment (yyd106) [inline deliveries/KliKli-milestone_1.md:8]: PR link is this one: https://github.com/crustio/Crust-Grants-Program/pull/24/files

File: deliveries/KliKli-milestone_1.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+Please provide `PR Link` and `Milestone Number` at following.
+* **PR Link:** Provide a link to the initial accepted pull request of your application to the [Crust Grants Program repository](https://github.com/crustio/Crust-Grants-Program). 
+* **Milestone Number:** The number of the milestone
+
+Please provide a list of all deliverables of the milestone extracted from the initial application and a link to the deliverable itself. Ideally all links inside the below table should include a commit hash, which will be used for testing. If you don't provide a commit hash, we will work off the default branch of your repository. Thus, if you plan on continuing work after delivery, we suggest you create a separate branch for either the delivery or your continuing work.
+
+
+| Number | Deliverable | PR Link | Notes |
+| ------------- | ------------- | ------------- |------------- |
+| 1. | Implement Crust Order Modules |https://github.com/mg00736448/Crust_KliKli/tree/main/app| Video upload interface and data recording；Call the CRUST network to place an order and update the status regularly |
```
Comment (yyd106) [inline deliveries/KliKli-milestone_1.md:16]: This repo has no readable document

**Comment by mg00736448** [comment]
update PR Link ，Please look over

File: deliveries/KliKli-milestone_1.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+Please provide `PR Link` and `Milestone Number` at following.
+* **PR Link:** Provide a link to the initial accepted pull request of your application to the [Crust Grants Program repository](https://github.com/crustio/Crust-Grants-Program). 
+* **Milestone Number:** The number of the milestone
+
+Please provide a list of all deliverables of the milestone extracted from the initial application and a link to the deliverable itself. Ideally all links inside the below table should include a commit hash, which will be used for testing. If you don't provide a commit hash, we will work off the default branch of your repository. Thus, if you plan on continuing work after delivery, we suggest you create a separate branch for either the delivery or your continuing work.
+
+
+| Number | Deliverable | PR Link | Notes |
+| ------------- | ------------- | ------------- |------------- |
+| 1. | Implement Crust Order Modules |https://github.com/crustio/Crust-Grants-Program/pull/24/files| Video upload interface and data recording；Call the CRUST network to place an order and update the status regularly |
```
Comment (yyd106) [inline deliveries/KliKli-milestone_1.md:16]: This is a wrong PR.

**Comment by mg00736448** [comment]
update PR Link

**Review by yyd106 — APPROVED** [review]

**Pull request merged**